pg_cjk_parser.c

```diff
@@ -2845,6 +2845,94 @@ mark_hl_words(HeadlineParsedText *prs, TSQuery query, int highlight,
 
 }
 
+/*
+ * patch_pg_atoi: convert string to integer
+ *
+ * allows any number of leading or trailing whitespace characters.
+ *
+ * 'size' is the sizeof() the desired integral result (1, 2, or 4 bytes).
+ *
+ * c, if not 0, is a terminator character that may appear after the
+ * integer (plus whitespace).  If 0, the string must end after the integer.
+ *
+ * Unlike plain atoi(), this will throw ereport() upon bad input format or
+ * overflow.
+ */
+int32
+patch_pg_atoi(const char *s, int size, int c)
```

Comment:
I think we should use `pg_strtoint32`
https://github.com/postgres/postgres/blob/611806cd726fc92989ac918eac48fd8d684869c7/src/backend/tsearch/wparser_def.c#L2640
instead of `patched_pg_atoi`

Reply:
I don't know С well. You are free to correct as you see fit.